index.js

```diff
@@ -345,6 +345,13 @@
     function(typeRep) { return []; }
   );
 
+  //  Ordering :: Type
+  var Ordering = $.EnumType(
+    'sanctuary/Ordering',
+    readmeUrl('Ordering'),
+    ['LT', 'EQ', 'GT']
+  );
```

Comment:
Ordering values set to -1, 0 and 1: https://github.com/sanctuary-js/sanctuary/pull/365/files#diff-168726dbe96b3ce427e7fedce31bb0bcR352

Things look a lot better now!